src/applications/dhp-connected-devices/tests/components/DeviceDisconnectionCard.unit.spec.jsx

```diff
@@ -41,9 +42,11 @@ describe('Device disconnection card', () => {
       expect(getDisconnectModal(screen)).to.not.exist;
     });
     it('Renders vendor terms and conditions', () => {
-      expect(screen.getByTestId('test-vendor-terms-and-conditions')).to.exist;
-      expect(screen.getByTestId('test-vendor-terms-and-conditions-content')).to
-        .exist;
+      const card = renderInReduxProvider(
+        <DeviceDisconnectionCard device={{ name: 'Fitbit', key: 'fitbit' }} />,
+      );
+      expect(card.getByTestId('fitbit-terms-and-conditions')).to.exist;
+      expect(card.getByTestId('fitbit-terms-and-conditions-content')).to.exist;
     });
```

Comment:
To test this fails when it's supposed to, remove `<TermsAndCondidtions` component in the `DeviceDisconnectionCard` component.

Reply:
QA: Task failed successfully